distributed/client.py

```diff
@@ -1702,8 +1701,7 @@ async def _close(self, fast=False):
 
         self.status = "closing"
 
-        for preload in self.preloads:
-            await preload.teardown()
+        await self.preloads.teardown()
```

Comment:
I cannot see an argument for raising here.

Reply:
Agreed; we should call logger.exception and continue